### PR TITLE
Feature/parent rewards

### LINKED ISSUE #49: Add functionality to delete chores and rewards

### PR DESCRIPTION
**Pull request by yixiaoliuw**
I have implemented the reward list for parent which displays all rewards. For rewards without having been redeemed, I display a delete button. The delete request should be working properly. I have also updated redeem rewards to display number of redemptions. I tried to add navigation functionality to the parent home fragment to the reward list activity but currently the navigation is not working. I simply set an onclick event for the added Reward List button in the fragment_home.

### CODE REVIEW

File: Android/ChoreCenterApp/app/src/main/java/com/cse403chorecenter/chorecenterapp/ui/home/HomeFragment.java [hunk 1]
```diff
@@ -78,6 +78,17 @@ public void onClick(View view) {
             }
         });
 
+        // Reward List
+        Button bAllRewards = (Button) view.findViewById(R.id.button_home_all_rewards);
+        bRewardHistory.setOnClickListener(new View.OnClickListener()
```
Comment (mllu777) [inline Android/ChoreCenterApp/app/src/main/java/com/cse403chorecenter/chorecenterapp/ui/home/HomeFragment.java]: should be bAllRewards

**Comment by yixiaoliuw** [comment]
Fixed the button issue.

**Review by mllu777 — APPROVED** [review]
Looks Good

**Review by yaretzj — APPROVED** [review]
Good stuff!

**Pull request merged**